spec/rails_helper.rb

```diff
@@ -12,7 +12,7 @@
 Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
 
 RSpec.configure do |config|
-  config.file_fixture_path = "#{::Rails.root}/spec/fixtures"
+  config.file_fixture_path = Rails.root.join("spec/fixtures").to_s
```

Comment:
Don't think the `to_s` is needed:

https://github.com/unboxed/bops/blob/main/spec/rails_helper.rb#L22-L23